app/sms/server/services/zenvia.js

```diff
@@ -0,0 +1,159 @@
+import { Meteor } from 'meteor/meteor';
+import * as zenvia from '@zenvia/sdk';
+import { Random } from 'meteor/random';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import filesize from 'filesize';
+
+import { settings } from '../../../settings';
+import { SMS } from '../SMS';
+import { Notifications } from '../../../notifications';
+import { fileUploadIsValidContentType } from '../../../utils/lib/fileUploadRestrictions';
+
+const MAX_FILE_SIZE = 5242880;
+
+const notifyAgent = (userId, rid, msg) => Notifications.notifyUser(userId, 'message', {
+	_id: Random.id(),
+	rid,
+	ts: new Date(),
+	msg,
+});
+
+class Zenvia {
+	constructor() {
+		this.apiToken = settings.get('SMS_Zenvia_API_Token');
+		this.channel = settings.get('SMS_Zenvia_Channel');
+		this.from = settings.get('SMS_Zenvia_From');
+		this.fileUploadEnabled = settings.get('SMS_Zenvia_FileUpload_Enabled');
+		this.mediaTypeWhiteList = settings.get('SMS_Zenvia_FileUpload_MediaTypeWhiteList');
+	}
+
+	parse(data) {
+		const defaultLanguage = settings.get('Language') || 'en';
+		let numMedia = 0;
+
+		const returnData = {
+			from: data.message.from,
+			to: data.message.to,
+			body: '',
+
+			extra: {
+				toCountry: undefined,
+				toState: undefined,
+				toCity: undefined,
+				toZip: undefined,
+				fromCountry: undefined,
+				fromState: undefined,
+				fromCity: undefined,
+				fromZip: undefined,
+				fromLatitude: undefined,
+				fromLongitude: undefined,
+			},
+		};
+		returnData.media = [];
+
+		if(data.message.contents.length>0){
+			for (let contentIndex = 0; contentIndex < (data.message.contents.length); contentIndex++) {
+				if(data.message.contents[contentIndex].type == 'text'){
```

Comment:
You can use a `Switch Statement` here avoiding the `if`/`else`:
```suggestion
		const content = contents[contentIndex];
		switch (content.type) {
			case 'text':
				...
				break
			case 'file':
				...
				break
```